tests/opentelemetry/test_config.py

```diff
@@ -0,0 +1,298 @@
+        "OTEL_TRACES_EXPORTER": "True",
+        "DD_TRACE_ENABLED": "True",
+        "OTEL_METRICS_EXPORTER": "none",
+        "DD_RUNTIME_METRICS_ENABLED": "True",
+        "OTEL_LOGS_EXPORTER": "warning",
+        "OTEL_RESOURCE_ATTRIBUTES": "deployment.environment=prod,service.name=bleh,"
+        "service.version=1.0,testtag1=random1,testtag2=random2,testtag3=random3,testtag4=random4",
+        "DD_TAGS": "env:staging",
+        "OTEL_SDK_DISABLED": "True",
+        "DD_TRACE_OTEL_ENABLED": "True",
+    },
+    err=b"Unsupported OTEL logs exporter value detected: warning. Only the 'none' value is supported.\n",
+)
+def test_dd_otel_mixed_env_configuration():
+    from ddtrace import config
+
+    assert config.service == "DD_service_test", config.service
+    assert config._debug_mode is False, config._debug_mode
+    assert config._propagation_style_extract == ["b3"], config._propagation_style_extract
+    assert config._trace_sample_rate == 1.0, config._trace_sample_rate
+    assert config._tracing_enabled is True, config._tracing_enabled
+    assert config._runtime_metrics_enabled is True, config._runtime_metrics_enabled
+    assert config.tags == {
+        "env": "staging",
+    }, config.tags
+    assert config.service == "DD_service_test"
+    assert config._otel_enabled is True, config._otel_enabled
+
+
+@pytest.mark.subprocess(
+    env={
+        "OTEL_SERVICE_NAME": "Test",
+        "OTEL_LOG_LEVEL": "debug",
+        "OTEL_PROPAGATORS": "jaegar, tracecontext, b3",
+        "OTEL_TRACES_SAMPLER": "always_off",
+        "DD_TRACE_SAMPLE_RATE": "1.0",
+        "OTEL_TRACES_EXPORTER": "OTLP",
+        "OTEL_METRICS_EXPORTER": "none",
+        "OTEL_LOGS_EXPORTER": "warning",
+        "OTEL_RESOURCE_ATTRIBUTES": "deployment.environment=prod,service.name=bleh,"
+        "service.version=1.0,testtag1=random1,testtag2=random2,testtag3=random3,testtag4=random4",
+        "OTEL_SDK_DISABLED": "False",
+    },
+    err=b"Following style not supported by ddtrace: jaegar.\n"
+    b"An unrecognized trace exporter 'otlp' is being used; setting DD_TRACE_ENABLED to False.\n"
+    b"Unsupported OTEL logs exporter value detected: warning. Only the 'none' value is supported.\n",
+)
+def test_dd_otel_missing_dd_env_configuration():
+    from ddtrace import config
+
+    assert config.service == "Test", config.service
+    assert config._debug_mode is True, config._debug_mode
+    assert config._propagation_style_extract == ["tracecontext", "b3"], config._propagation_style_extract
+    assert config._trace_sample_rate == 1.0, config._trace_sample_rate
+    assert config._tracing_enabled is False, config._tracing_enabled
+    assert config._runtime_metrics_enabled is False, config._runtime_metrics_enabled
+    assert config.tags == {
+        "env": "prod",
+        "testtag1": "random1",
+        "testtag2": "random2",
+        "testtag3": "random3",
+        "testtag4": "random4",
+    }, config.tags
+    assert config.service == "Test"
```

Comment:
This is checked twice in the same test. We need to make sure we are not doing this in multiple tests
```suggestion
```